### PR TITLE
Moved messages to amp-user-notification

### PR DESCRIPTION
**Pull request by andreban**

### CODE REVIEW

**Comment by andreban** [comment]
fixes #26

File: controllers/amp-user-notification/api.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+var express = require('express');
+var router = express.Router();
+var UserNotification = require("../../models/user-notifications");
+
+/**
+ * AMP will make a CORS GET request to this URL to determine whether 
+ * the notification should be shown. We will append the elementId and 
+ * ampUserId query string fields to the href provided on the data-show-if-href
+ * attribute.
```
Comment (sebastianbenz) [inline controllers/amp-user-notification/api.js]: What do you mean by "We will append the elementId and ampUserId query string fields to the href provided on the data-show-if-href attribute."?

File: controllers/amp-user-notification/api.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+router.get('/show', function(req, res){
+	var userId = req.query.ampUserId;
+	var notificationId = req.query.elementId;
+	var showNotification = UserNotification.shouldShowNotification(notificationId, userId);
+	res.json({ "showNotification": showNotification});
+});
+
+/**
+ * AMP will make a CORS POST request to this URL transmitting the elementId 
+ * and ampUserId only when the user has explicitly agreed.
```
Comment (sebastianbenz) [inline controllers/amp-user-notification/api.js]: Rephrase "when the user has explicitly agreed" to "when the user dismisses the notification"

File: models/user-notifications.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+exports.shouldShowNotification = function(notificationId, userId) {
+	var user = DISMISSED_NOTIFICATIONS[userId];
+	if (!user) {
+		return true;
+	}
+	return !user[notificationId];
+}
+
+/**
+ * Dismisses further notifications for the user and tnotificationId
```
Comment (sebastianbenz) [inline models/user-notifications.js]: typo in tnotification

**Comment by sebastianbenz** [comment]
Would be great if you could fix the comments.

File: controllers/amp-user-notification/api.js (hunk 2 above)
Comment (andreban) [inline controllers/amp-user-notification/api.js]: Done

File: models/user-notifications.js (hunk 3 above)
Comment (andreban) [inline models/user-notifications.js]: Done

File: controllers/amp-user-notification/api.js (hunk 1 above)
Comment (andreban) [inline controllers/amp-user-notification/api.js]: Change to "The AMP runtime will append..."

**Comment by sebastianbenz** [comment]
👍🏻

File: views/amp-access/article.html [hunk 4]
```diff
@@ -78,27 +79,38 @@
...
     <main role="main">
     <article>
-      <!-- 
-        This section is only visible by subscribers. The subscriber variable is 
-        defined in the authentification response data.
-      -->
-      <section amp-access="access AND subscriber" amp-access-hide class="meter-section" data-z-depth="2">
-        Thanks for being a subscriber. You rock!
-      </section>
+      <amp-user-notification
```
Comment (dvoytenko) [inline views/amp-access/article.html]: So, this is actually a bit problematic at this time. We don't have a good integration with amp-user-notifications yet. The biggest issue is that once dismissed, the notification will not come back and there are other inconsistencies. If we need it only for "dismiss" - it's easy enough to simply put a button inside of the metered section with `on:hide` action:

```
<a on="tap:meter-notif.hide">[Close]</a>
```

**Pull request merged**